backend/src/data/repository/user.ts

```diff
@@ -44,6 +40,46 @@ export class UserRepository {
     return data;
   }
 
+  async getUsers(): Promise<DTO.IUserDoc[] | null> {
+    const data = await User.find(
+      {},
+      { firstName: 1, lastName: 1, photo: 1, birthday: 1, country: 1, speak: 1, learn: 1 }
+    );
+    return data;
+  }
+
+  // Search
+
+  async getUsersByParams(params: DTO.FilterRequest): Promise<DTO.IUserDoc[] | null> {
+    const { name, birthdateFrom, birthdateTo, sex, country, language, level } = params;
+    const data = await User.find({
+      $and: [
+        {
+          $expr: {
+            $function: {
+              body: `function(firstName, lastName) {
```

Comment:
body: `(firstName, lastName) => ((firstName + ' ' + lastName).toLowerCase().includes('${name.trim().toLowerCase()')

Reply:
I'll keep no-arrow function, because it doesn't want to work